stubs/WeakReference.stub

```diff
@@ -16,3 +16,14 @@ final class WeakReference
 	/** @return ?T */
 	public function get() {}
 }
+
+
+/**
+ * @template TKey of object
+ * @template TValue
+ * @implements \ArrayAccess<TKey, TValue>
+ * @implements \IteratorAggregate<TKey, TValue>
+ */
+final class WeakMap implements \ArrayAccess, \Countable, \IteratorAggregate
```

Comment:
The class should also be added here: https://github.com/phpstan/phpstan-src/blob/39aa45feab3b7a0f0945bc65f8daccd5cda5d2b0/conf/config.neon#L39-L41

Reply:
Why is that? I'm not understanding the requirement here.